client/src/main/com/sinch/sdk/domains/conversation/adapters/ContactService.java

```diff
@@ -26,26 +26,27 @@
 import com.sinch.sdk.domains.conversation.models.requests.GetChannelProfileByChannelRequestParameters;
 import com.sinch.sdk.domains.conversation.models.requests.GetChannelProfileByContactRequestParameters;
 import com.sinch.sdk.domains.conversation.models.responses.ContactListResponse;
-import com.sinch.sdk.models.Configuration;
+import com.sinch.sdk.models.ConversationContext;
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Map;
 
 public class ContactService implements com.sinch.sdk.domains.conversation.ContactService {
 
-  private final Configuration configuration;
+  private final String uriPathID;
```

Comment:
The same variable is named `uriUUID` on the main branch for other APIs. Which one will win?

Reply:
😕 both exists (for SMS)...
refactored to use `uriUUID` for Conversation (not refactored for SMS: will have to be adressed outside of this feature dedicated branch)